features/step_definitions/checks_for_vulnerable_gems_steps.rb

```diff
@@ -1,11 +1,14 @@
 def executing_gem_freshness
-  "executing 'bundle-audit update; (cd . && bundle-audit check)'"
+  "executing 'bundle-audit update &> /dev/null && (cd . && bundle-audit check)'"
 end
 
 def gem_freshness_output
   if @projectname
     prep = "cd spec/fixtures/#{@projectname}/ && cp Gemfile_lock Gemfile.lock"
-    `#{prep} && bundle-audit check; rm -f Gemfile.lock`
+    `#{prep} && bundle-audit check; rm -f Gemfile.lock`.
+      split("\n").
+      reject { |line| line.match('patched versions found') }.
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.